index.js

```diff
@@ -179,7 +180,7 @@ module.exports = {
 
     var ui = this.ui;
 
-    return jshintTrees(tree, {
+    return jshintTrees(new Funnel(tree, { include: [/\.js$/] }), {
```

Comment:
Wow, quick on that one ;)

Are globs generally more performant than regexps with broccoli-funnel?